### ISSUE TITLE
Running bor without heimdall ?

### ISSUE THREAD

**Issue by morazzela** [maticnetwork/bor#165]
Hey !

I'm trying since a few days to setup bot without running heimdall, bor is syncing well up to ~95%, but then I get "BAD BLOCK" error.
I'm using the "--bor.withoutheimdall" option, also running bor in "full" mode.
Searched in the issues and tried to rewind the blockchain to -100 blocks, -1000 blocks using the "debug.setHead("0x...")" command but nothing worked (temporarily at least, getting the error few hours later). 

Is it because I need to run an heimdall instance and connect bor to heimdall ?

**Comment by ssandeep** [comment]
Yes, if you want to run a full node, you will have to run both heimdall as well as bor. The `--bor.withoutheimdall` option is for testing purpose only.
